packages/web-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -19,7 +19,10 @@ const Dropdown = (props: SpiritDropdownProps) => {
     children,
     renderTrigger,
     enableAutoClose,
+    /* Deprecation (https://jira.lmc.cz/browse/DS-493) --> */
```

Comment:
Please use the standard JSDoc `@deprecated` notice: https://jsdoc.app/tags-deprecated.html.